### ISSUE TITLE
Issue with Domainmod 4.20.01

### ISSUE THREAD

**Issue by prbt2016** [domainmod/domainmod#162]
Hello ,

I was in the process of manual installation of Domainmod 4.20.01 on Centos7 with PHP 8.1/8.2 , MYSQL 5.7, Apache 2.4 .

However , I get blank page after entering all the details like language selection, Currency , Timezone , admin email and system email . 

So after turning display_errors = On in php.ini  still there wasn't any error, not even in apache logs, 
so I added this debug code on top of go.php i.e 

```
function shutdown()	{ 
  print_r(error_get_last());	
}
	
register_shutdown_function('shutdown');
```

And re-ran the install, I get the following error shown i.e : 

```

Array ( [type] => 1 [message] => Uncaught Error: Class "Webmozart\Assert\Assert" not found in /{{PATH}}/{{TO}}/{{DOMAINMOD}}/vendor/dragonmantank/cron-expression/src/Cron/CronExpression.php:202 Stack trace: #0 /{{PATH}}/{{TO}}/{{DOMAINMOD}}/vendor/dragonmantank/cron-expression/src/Cron/CronExpression.php(187): Cron\CronExpression->setExpression('0 7 * * *') #1 /{{PATH}}/{{TO}}/{{DOMAINMOD}}/install/go.php(1083): Cron\CronExpression->__construct('0 7 * * *') #2 {main} thrown [file] => /{{PATH}}/{{TO}}/{{DOMAINMOD}}/vendor/dragonmantank/cron-expression/src/Cron/CronExpression.php [line] => 202 )

```

What could be the issue ?. Could you please replicate this at your end .

**Comment by prbt2016** [comment]
Hello ,

Getting same issue in 4.20.05 as well . Checked apache logs as well , It's throwing the same issue, and with debug code i.e : 

```
[{{DATE}}] [error] [client {{IPADDRESS}}] PHP Fatal error:  Uncaught Error: Class "Webmozart\\Assert\\Assert" not found in /{{PATH}}/{{TO}}/{{DOMAINMOD}}/vendor/dragonmantank/cron-expression/src/Cron/CronExpression.php:202\nStack trace:\n#0 /{{PATH}}/{{TO}}/{{DOMAINMOD}}/vendor/dragonmantank/cron-expression/src/Cron/CronExpression.php(187): Cron\\CronExpression->setExpression('0 7 * * *')\n#1 /{{PATH}}/{{TO}}/{{DOMAINMOD}}/install/go.php(1085): Cron\\CronExpression->__construct('0 7 * * *')\n#2 {main}\n  thrown in /{{PATH}}/{{TO}}/{{DOMAINMOD}}/vendor/dragonmantank/cron-expression/src/Cron/CronExpression.php on line 202, referer: 

```

Could you please check and resolve this issue ?.

**Comment by chetcuti** [comment]
This is resolved in v4.20.06, which was released earlier tonight.

Long story short, Webmozart is required by one of DomainMOD's dependencies, but in the past it was optional, and at one point I excluded this optional dependency using git in order to keep DomainMOD as small as I could. That exclusion was still in place, but since Webmozart has now changed to a required dependency, that kind of broke things. I've removed the exclusion now and Webmozart is included in DomainMOD, so this issue should now be resolved.
